src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -194,7 +195,29 @@ def _write_sbatch_script(
             f"#SBATCH -N {args['num_nodes']}",
         ]
 
+        self._append_sbatch_directives(batch_script_content, args, output_path)
+
+        batch_script_content.extend([env_vars_str, "", srun_command])
+
+        batch_script_path = output_path / "cloudai_sbatch_script.sh"
+        with batch_script_path.open("w") as batch_file:
+            batch_file.write("\n".join(batch_script_content))
+
+        return f"sbatch {batch_script_path}"
+
+    def _append_sbatch_directives(
+        self, batch_script_content: List[str], args: Dict[str, Any], output_path: Path
+    ) -> None:
+        """
+        Append SBATCH directives to the batch script content.
+
+        Args:
+            batch_script_content (List[str]): The list of script lines to append to.
+            args (Dict[str, Any]): Arguments including job settings.
+            output_path (str): Output directory for script and logs.
```

Comment:
The docstring is outdated (str -> Path).

Reply:
updated it.